lib/restaurant.rb

```diff
@@ -0,0 +1,27 @@
+require 'order'
```

Comment:
Think this line is causing an error. Prevents me from running the program locally in IRB. In this case being explicit with the dependent file location with `require relative` would be good.